eawork/migrations/0020_add_new_company_fields.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 3.2.15 on 2022-10-07 13:31
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('eawork', '0019_set_all_jobs_to_80k_importable'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='company',
+            name='forum_url',
+            field=models.URLField(blank=True, max_length=511, verbose_name='Forum'),
+        ),
+        migrations.AddField(
+            model_name='company',
+            name='is_top_recommended_org',
+            field=models.BooleanField(default=False),
+        ),
+    ]
```

Comment:
oh btw you can run linting as `black . && isort .`

I can add a git commit hook latter

Reply:
oh, sick